addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/data/TankerkoenigService.java

```diff
@@ -0,0 +1,82 @@
+package org.openhab.binding.tankerkoenig.internal.data;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+
+import org.apache.commons.io.IOUtils;
+import org.openhab.binding.tankerkoenig.internal.config.TankerkoenigListResult;
+import org.openhab.binding.tankerkoenig.internal.serializer.CustomTankerkoenigListResultDeserializer;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+/***
+ * Serivce class requesting data from tankerkoenig api and providing result objects
+ *
+ * @author Dennis Dollinger
+ *
+ */
+public class TankerkoenigService {
+
+    public TankerkoenigListResult GetTankstellenListData(String apikey, String locationIDs) {
+        TankerkoenigListResult result = this.GetTankerkoenigListResult(apikey, locationIDs);
+        return result;
+    }
+
+    String response = "";
```

Comment:
Please define it as local variable in the methods tha return a response value or remove (because I don't think you need it).